src/test/java/com/networknt/schema/utils/DefaultTest.java

```diff
@@ -0,0 +1,38 @@
+package com.networknt.schema.utils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.JsonSchemaRef;
+import com.networknt.schema.SpecVersion.VersionFlag;
+public class DefaultTest {
+    
+    @Test
+    void testGetDefaultNodeNotNull() throws Exception {
```

Comment:
This test case is exactly the same as the `testGetDefaultNodeEquals` case you have so you should remove this. If you want you can move the `assertNotNull` here to that other test case, but I don't think that is needed.

You should test
* The behavior when the node cannot be found
* The behavior when there is a `$ref`

Reply:
added node not found test case

Reply:
Still need the test for `$ref`

Reply:
done